src/ToricVarieties/ToricDivisorClasses/constructors.jl

```diff
@@ -0,0 +1,208 @@
+#########################
+# 1: The Julia type for ToricDivisorClasses
+#########################
+
+@attributes mutable struct ToricDivisorClass
+            toric_variety::AbstractNormalToricVariety
```

Comment:
That's a weird indentation; we usually do 2/3/4 spaces, but this looks like it tries to align things with the start of the word "mutable" which is unusual